src/acceptheaderpayee/handler.js

```diff
@@ -0,0 +1,449 @@
+    } catch (err) {
+      Logger.error(err)
+      histTimerEnd({ success: false, fsp: 'payee', operation: 'getPartiesByTypeAndId', source: req.headers['fspiop-source'], destination: req.headers['fspiop-destination'] })
+    }
+  })()
+
+  return h.response().code(202)
+}
+
+exports.postQuotes = function (req, h) {
+  (async function () {
+    const histTimerEnd = Metrics.getHistogram(
+      'sim_request',
+      'Histogram for Simulator http operations',
+      ['success', 'fsp', 'operation', 'source', 'destination']
+    ).startTimer()
+
+    const metadata = `${req.method} ${req.path}`
+    const quotesRequest = req.payload
+    Logger.info((new Date().toISOString()), ['IN PAYEEFSP::'], `received: ${metadata}. `)
+    Logger.info(`incoming request: ${quotesRequest.quoteId}`)
+
+    // Saving Incoming request
+    const incomingRequest = {
+      headers: req.headers,
+      data: req.payload
+    }
+    requests.set(quotesRequest.quoteId, incomingRequest)
+
+    const quotesResponse = {
+      transferAmount: {
+        amount: quotesRequest.amount.amount,
+        currency: quotesRequest.amount.currency
+      },
+      payeeFspFee: {
+        amount: '1',
+        currency: quotesRequest.amount.currency
+      },
+      payeeFspCommission: {
+        amount: '1',
+        currency: quotesRequest.amount.currency
+      },
+      expiration: new Date(new Date().getTime() + 10000),
+      ilpPacket: transfersIlpPacket,
+      condition: transfersCondition
+    }
+
+    try {
+      const url = quotesEndpoint + '/quotes/' + quotesRequest.quoteId
+      const protectedHeader = {
+        alg: 'RS256',
+        'FSPIOP-Source': `${req.headers['fspiop-destination']}`,
+        'FSPIOP-Destination': `${req.headers['fspiop-source']}`,
+        'FSPIOP-URI': `/quotes/${quotesRequest.quoteId}`,
+        'FSPIOP-HTTP-Method': 'PUT',
+        Date: ''
+      }
+      const fspiopSignature = {
+        signature: signature,
+        protectedHeader: `${base64url.encode(JSON.stringify(protectedHeader))}`
+      }
+      const opts = {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/vnd.interoperability.quotes+json;version=1.0',
+          'FSPIOP-Source': 'acceptheaderpayeefsp',
+          'FSPIOP-Destination': req.headers['fspiop-source'],
+          Date: new Date().toUTCString(),
+          'FSPIOP-Signature': `${JSON.stringify(fspiopSignature)}`,
+          'FSPIOP-HTTP-Method': 'PUT',
+          'FSPIOP-URI': `/quotes/${quotesRequest.quoteId}`,
+          traceparent: req.headers.traceparent ? req.headers.traceparent : '',
+          tracestate: req.headers.tracestate ? req.headers.tracestate : ''
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        }),
+        data: JSON.stringify(quotesResponse)
+      }
+      Logger.info((new Date().toISOString()), 'Executing PUT', url)
+      const res = await request(url, opts)
```

Comment:
I dont believe that this would work. axios requires the url to be sent in the opts. therefore your call should look like 
```
const opts = {
    url: '',
    method: 'PUT',
    headers: [],
    ....
}
const res = await request(opts)
```